packages/algob/src/internal/deployer_cfg.ts

```diff
@@ -35,7 +35,7 @@ export class DeployerConfig {
     this.runtimeEnv = runtimeEnv;
     this.cpData = loadCheckpointsRecursive();
     this.algoOp = algoOp;
-    this.accounts = mkAccountIndex(runtimeEnv.network.config.accounts);
+    this.accounts = mkAccountIndex(runtimeEnv.network.config.accounts ?? []);
```

Comment:
do we not need accounts in purestake api ?

Reply:
This is to handle a case if no accounts are passed in config.